client/components/Landing.tsx

```diff
@@ -1,12 +1,13 @@
-import { Image, Heading, Container, Text, Link } from "@chakra-ui/react";
+import { Image, Heading, Box, Text, Link } from "@chakra-ui/react";
 
+import NextLink from 'next/link'
```

Comment:
I've just started learning NextJS a few days ago. I can't seem to find `NextLink` anywhere in the documentation. How is it different from the regular ol' `Link` ? Could you point me in the right direction? 🙏

Reply:
There's no real difference here. I'm using `NextLink` to identify that it's from NextJS and not `Link` from ChakraUI

Reply:
> How is it different from the regular ol' `Link` ?

The main difference is that `Link` from `next/link` intercepts user interaction to display the correct page component. Since this is [a single-page application](https://en.wikipedia.org/wiki/Single-page_application), the browser shouldn't actually navigate to another page. 

If we use `Link` from `@chakra-ui/react`, it doesn't intercept user interaction so clicking a link causes the page to fully reload. 

The api reference for `next/link` also [provides a good idea of all the things it handles](https://nextjs.org/docs/api-reference/next/link).

Reply:
Thanks for that. I misunderstood it was using ChakraUI `Link` and not the NextJS `Link` - It makes sense now 🤙🤙